auth-web/src/components/auth/account-settings/transaction/Transactions.vue

```diff
@@ -188,13 +188,6 @@ export default defineComponent({
     })
 
     const credit = ref(0)
-
-    const isTransactionsAllowed = computed((): boolean => {
```

Comment:
no longer needed. Permissions will hide/show this based on 'transaction_history' permission, this has been updated for the auth-api portion. API blocks if there are no appropriate security roles.

Reply:
Looks like it was originally a work around for different access points, added the redirect back in, but updated to check permissions